ClientSide(FrontEnd)/clay_/src/components/forgot.css

```diff
@@ -0,0 +1,87 @@
+.start{
+    display: flex;
+    margin-top: -2%;
+    justify-content: center;
+    border-radius: 70px;
+}
+.start1{
+    padding: 30px;
+    width: 550px;
+    margin-top: -490px;
+    border-radius: 70px;
+    background-color: #FFB5C3;
+    background-image: url('/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/Screenshot from 2023-06-06 16-50-35.png');
+}
+.starting2{
+    padding: 30px;
+    width: 400px;
+    height: 350px;
+    border-radius: 70px;
+}
+.ame{
+    margin: 20px;
+    padding: 15px 40px;
+    margin-left: 60px;
+    border-radius: 30px;
+    background-color: white;
+    border: 2px solid white;
+
+}
+.sername{
+    margin: 20px;
+    padding: 15px 40px;
+    margin-left: 60px;
+    border-radius: 30px;
+    background-color: white;
+    border: 2px solid white;
+}
+.mail{
+    margin: 20px;
+    padding: 15px 40px;
+    margin-left: 60px;
+    border-radius: 30px;
+    background-color: white;
+    border: 2px solid white;
+}
+.umber{
+    margin: 20px;
+    padding: 15px 40px;
+    margin-left: 60px;
+    border-radius: 30px;
+    background-color: white;
+    border: 2px solid white;
+}
+.assword{
+    margin: 20px;
+    padding: 15px 40px;
+    margin-left: 60px;
+    border-radius: 30px;
+    background-color: white;
+    border: 2px solid white;
+}
+.sub{
+    padding:15px 30px;
+    /* margin-left: 130px; */
+    margin: 10px 0px 0px 130px;
+    border-radius: 30px;
+    background-color: white;
+    border: 2px solid white;
+}
+.pic1{
+    margin-left: 1000px;
+}
+/* .image3{
+    /* padding: 30px; */
+    /* width: 460px;
+    height: 567px; */
+    /* margin: 30px; */
+    /* margin-top: -30px; */
+    /* border: 2px solid green; */
+    /* margin-left: -30px; */
+    /* margin-top: -300px; */
+    /* border-radius: 70px; */
+/* } */ 
+.h12{
+    /*      text-align: center; */
+    margin-left: 80px;
+}
```

Comment:
add one line of empty space at the end of your code

Reply:
i added that empty space